examples/exporting_for_inference.ipynb

```diff
@@ -23,11 +23,11 @@
     "\n",
     "We'll touch on:\n",
     "\n",
-    "* [Our standalone export API](#torchscript-export-using-standalone-api)\n",
-    "* [Exporting from the trainer with callbacks](#export-using-a-callback)\n",
-    "* [Exporting from the trainer directly](#alternative-export)\n",
-    "* [Exporting from an existing checkpoint](#exporting-from-an-existing-checkpoint)\n",
-    "* [Supported Composer algorithms](#algorithm-compatibility)\n",
+    "* [Our standalone export API](#Torchscript-Export-Using-Standalone-API)\n",
```

Comment:
This one still doesn't work because it automatically transforms it  from`#Torchscript-Export-Using-Standalone-API` to `#Torchscript-Export-Using-Standalone-Api` in the URL

Reply:
at least when I test it using
```bash
make html
make host
```

Reply:
ok works now